Include fan-out operators in the operator index

raboof opened the pull request:
Also helps resolve https://github.com/akka/akka.github.com/issues/606

raboof committed:
Include fan-out operators in the operator index

Also helps resolve https://github.com/akka/akka.github.com/issues/606

helena commented on akka-docs/src/main/paradox/stream/operators/index.md:
```diff
@@ -257,6 +257,22 @@ the inputs in different ways.
+These have one input and multiple outputs. They might route the elements between different outputs, or emit elements on
+multiple outputs at the same time.
+
+There is currently no 'fluent' API available for the fan-out operators, to use them you will have to use the @ref[Graph DSL](../stream-graphs.md#constructing-graphs).
```
```suggestion
A 'fluent' API for the fan-out operators is not yet available, until then use the @ref[Graph DSL](../stream-graphs.md#constructing-graphs).
```

helena reviewed (approved):
One minor doc suggestion, otherwise LGTM

johanandren commented on akka-docs/src/main/categories/fan-out-operators.md:
```diff
@@ -1,2 +1,4 @@
\ No newline at end of file
+multiple outputs at the same time.
+
+There is currently no 'fluent' API available for the fan-out operators, to use them you will have to use the @ref[Graph DSL](../stream-graphs.md#constructing-graphs).
```
Actually, at few of them, or special cases of them, do have 'fluent' API: `alsoTo`, `divertTo` and `wireTap`.

Not sure if that is worth mentioning.

raboof replied:
These are currently filed under 'Simple operators'. I agree 'Fan-out operators' is more specific and thus better, I'll move them

raboof committed:
Update akka-docs/src/main/paradox/stream/operators/index.md

Co-Authored-By: Helena Edelson <helena@users.noreply.github.com>

raboof committed:
Move divertTo, alsoTo and wireTap to fan-out operators

raboof committed:
Fix checked-in generated doc

Still nice to have there, also makes it easy to see the effect of a change in
the generation when looking at the PR

johanandren commented:
Should it be backported @raboof ?

raboof commented:
I don't think it needs to
